src/climate_data/generate/scenario_daily.py

```diff
@@ -199,21 +197,6 @@ def generate_scenario_daily_main(  # noqa: PLR0912, PLR0915, C901
     print(f"{vid}: computing anomaly")
     v_anomaly = compute_anomaly(sref, target, anomaly_type)
 
-        # key = f"{len(v_anomaly.latitude)}_{len(v_anomaly.longitude)}"
-
-        # Is this no longer necessary because each variant is being handled independently?
```

Comment:
I removed the commented out code, but this question is still to be answered:

Is this section no longer necessary because each variant is being handled independently?

Reply:
Yeah, we don't need this when we hand variants independently.  This was about making memory usage more stable when we had a highly variable number of ensemble members.